index.bs

```diff
@@ -214,18 +228,35 @@ Modifications to other specifications {#mod}
 
 ### HTML: <a>event loop definitions</a> ### {#html-event-loop-dfn}
 
-Each task gets an associated <dfn for="task">start time</dfn>, <dfn for="task">end time</dfn>, and <dfn for="task">script evaluation environment settings object set</dfn>.
+Each task gets an associated <dfn for="task">script evaluation environment settings object set</dfn>.
 
 ### HTML: <a>event loop processing model</a> ### {#html-event-loop-processing}
 
```

Comment:
You need to add something like "each event loop has two associated values, event loop begin and event loop end, which are initially unset". Then references to them should use links, not `<var>`.

Otherwise they're algorithm-local variables which won't survive multiple invocations of the event loop processing model.

Reply:
Done